Snake/BusinessLayer/GameSnake.cs

```diff
@@ -8,15 +8,14 @@ namespace BusinessLayer
 {
     public class GameSnake
     {
-        enum Direction {right, down, left, up};
-        Direction directionFacing;
+        direction directionFacing;

```

Comment:
This doesn't seem like it's being used for anything right now, so you could probably remove it for the time being.